src/utils/Balance.tsx

```diff
@@ -367,35 +366,50 @@ const useBalanceHook = () => {
         resolve(balanceToString(balance, opts)),
       )
     },
-    [coinGeckoPrices, convertToCurrency, currency],
+    [convertToCurrency, currency, tokenPrices],
   )
 
   const toCurrencyString = useCallback(
-    (balance?: Balance<NetworkTokens | TestNetworkTokens>): Promise<string> => {
+    (
+      balance?: Balance<NetworkTokens | TestNetworkTokens>,
+      ticker: Ticker = 'HNT',
+    ): Promise<string> => {
       const defaultResponse = new Promise<string>((resolve) => resolve(''))
-      if (!balance || balance?.floatBalance === undefined) {
-        return defaultResponse
+
+      let bal = new Balance(
+        0,
+        ticker === 'SOL' ? CurrencyType.solTokens : CurrencyType.networkToken,
```

Comment:
Do we have any const we can read from for `SOL` and `HNT`?

Reply:
We do let me use `TICKER`

Reply:
Ticker is a discriminated union, so should be good. You could also use `Balance.fromIntAndTicker(0, ticker)`